app/assets/stylesheets/categories_items.scss

```diff
@@ -24,34 +27,45 @@
     }
     .item-info {
       margin: 10px;
-      aside {
-        display: inline-block;
-        width: 24%;
-      }
+      display: inline-flex;
+      flex-flow: row nowrap;
+      justify-content: space-between;
+      align-items: center;
       .item-title {
-        width: 44%;
+        order: 1;
+        width: 33%;
       }
-      .item-quantity input {
-        width: 44%;
-        float: right;
-        border: 1px solid $black;
+      .item-quantity {
+        order: 3;
+        width: 10%;
+        input {
+          width: 100%;
+          border: 1px solid $black;
+        }
       }
       .item-price {
-        width: 24%;
+        order: 2;
+        width: 30%;
+        font-size: 90%;
       }
     }
     .item-description {
       margin: 10px;
     }
     .item-card-button {
       text-align: center;
-      button {
+      margin: 0 auto;
+      form input[type=submit] {
+        padding: 10px 0;
         margin-bottom: 10px;
         width: 95%;
-        background: $lightbrown;
-        color: $black;
         border: 1px transparent;
         border-radius: 5px;
+        background: $lightbrown;
+        color: $black;
+      }
+      form input[type=submit]:hover {
```

Comment:
Good to keep in mind, but not sure how to style the button itself in a different way.